gammapy/maps/wcsmap.py

```diff
@@ -203,6 +203,8 @@ def to_hdulist(self, hdu=None, hdu_bands=None, sparse=False, format="gadf"):
 
         hdu_out = self.to_hdu(hdu=hdu, hdu_bands=hdu_bands, sparse=sparse)
 
+        # TODO: make this serialisable
+        self.meta.pop("livetime", None)
```

Comment:
Why is this needed? If this is required because of the defined format, it might be safer to copy the meta `dict` and remove the livetime entry in the copy.

Reply:
Yes, indeed modifying the meta dict on write is a bad idea, I‘ll fix this...

Reply:
I added a custom json serialiser, that handles `Quantity` objects. This is maybe useful in another context as well...